test/integration/selenium-helpers.js

```diff
@@ -8,9 +8,9 @@ const chromedriverVersion = require('chromedriver').version;
 
 const headless = process.env.SMOKE_HEADLESS || false;
 const remote = process.env.SMOKE_REMOTE || false;
-const ci = process.env.CI || false;
-const usingCircle = process.env.CIRCLECI || false;
-const buildID = process.env.CIRCLE_BUILD_NUM || '0000';
+const ciBuildPrefix = process.env.CI ?
+    `CI #${process.env.GITHUB_RUN_ID}/${process.env.GITHUB_RUN_ATTEMPT}` :
+    ''; // no prefix if not in CI
```

Comment:
This is the label that will appear in the Sauce Labs test browser when running remote tests.